go/pkg/pwdb/errors.go

```diff
@@ -3,13 +3,13 @@ package pwdb
 import (
 	"errors"
 
-	"github.com/jinzhu/gorm"
+	"gorm.io/gorm"
 	"pathwar.land/pathwar/v2/go/pkg/errcode"
 )
 
 func IsRecordNotFoundError(err error) bool {
-	return gorm.IsRecordNotFoundError(err) ||
-		gorm.IsRecordNotFoundError(errors.Unwrap(err))
+	return errors.Is(err, gorm.ErrRecordNotFound) ||
+		errors.Is(errors.Unwrap(err), gorm.ErrRecordNotFound)
```

Comment:
@remyoster We don't need to unwrap the err this is done by  errors.Is

Reply:
Good call, thanks ! I will fix it